specification/components/r4/schemas/queryParameters/Identifier.yaml

```diff
@@ -1,7 +1,7 @@
 name: identifier
 in: query   
 description: |
-  The UBRN which identifies the referral or advice and guidance.
+  The UBRN used to search for `ServiceRequest` resources.
```

Comment:
Might be worth highlighting what UBRN stands for here, I think it's Unique Booking Reference Number but it might be worth confirming that with the BAs.

Reply:
We refer to UBRN in the OAS docs as Unique Booking Reference Number, so I'll use that 👍